setup.cfg

```
@@ -30,3 +30,6 @@ ignore_errors = True
 
 [mypy-tests.*]
 ignore_errors = True
+
+setup_requires =
+    setuptools>=57.0,<57.5
```

Shouldn't this be in a `[options]` section
```suggestion

setup_requires =
    setuptools>=57.0,<57.5
```
But it looks like the project uses poetry to build packages. So I guess they won't accept this change.
Same for the `setup.py`.

---

Yes, I forgot to remove it. I wanted to test something quickly to have a `pip install -e .`. Je ne sais pas si il y a un équivalent avec poetry.